src/error_estimation/adjoint_refinement_estimator.C

```diff
@@ -367,6 +403,26 @@ void AdjointRefinementEstimator::estimate_error (const System & _system,
         }
     }
 
+  // We are all done with Dirichlet lift vectors and they should be removed, lest we run into I/O issues later
```

Comment:
Should we remove them, or just set_vector_preservation(false) here?  If we're likely to use them later, and we are, then there's no point in churning memory, right?

Reply:
Good point, they may get used later for user defined QoI evaluations.

Reply:
Reading in breaks if the lift vectors are not removed.